package.json

```diff
@@ -110,6 +118,7 @@
     "style-loader": "^3.3.2",
     "terser": "^5.17.3",
     "typescript": "^5.0.4",
+    "ua-parser-js": "^1.0.35",
```

Comment:
Any chance we can avoid adding an additional NPM dependency? A single guy operates this package.

Reply:
I didn't really want to reinvent the wheel and parse user-agent header on my own. This package will not affect our bundle size as we're using it server-side only. The rest of the usages come from `@types/ua-parser-js` package which also doesn't increase bundle size since it's stripped out during build process.